### ISSUE TITLE
@continuum-boot 'on' causes @continuum-restore 'on' to restore only one session

### ISSUE THREAD

**Issue by caleskog** [tmux-plugins/tmux-continuum#140]
I don't really know what causes this or if it could be another setting. But I have had a problem for some days now when booting my computer; only one session is restored when Tmux starts. After searching the internet and finding nothing, I just today stumbled upon a solution from a Reddit post from 2 years ago. The solution was to comment out _set -g @continuum-boot 'on'_.
And now everything works fine.

btw, I run Fedora Linux 40 with KDE Plasma and konsole as my terminal.

TLDR: comment out _set -g @continuum-boot 'on'_ as it seems to cause problems with the restoration of sessions.

**Comment by rpatterson** [comment]
Thanks for the tip! I removed `set -g @continuum-boot 'on'` and tried to get the `~/.config/systemd/user/tmux.service` systemd unit working without it but I discovered that `tmux-continuum` was repeatedly disabling that unit.  All told, I've decided to remove `tmux-continuum` all together and do my own minimal, less magical, automation.
